src/simplewallet/simplewallet.cpp

```diff
@@ -2138,13 +2138,16 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
   }
 
   vector<cryptonote::tx_destination_entry> dsts;
+  bool is_onetime = false;
   for (size_t i = 0; i < local_args.size(); i += 2)
   {
     cryptonote::tx_destination_entry de;
     bool has_payment_id;
     crypto::hash8 new_payment_id;
-    if (!tools::dns_utils::get_account_address_from_str_or_url(de.addr, has_payment_id, new_payment_id, m_wallet->testnet(), local_args[i]))
+    bool is_onetime_temp;
+    if (!tools::dns_utils::get_account_address_from_str_or_url(de.addr, has_payment_id, new_payment_id, is_onetime_temp, m_wallet->testnet(), local_args[i]))
       return true;
+    is_onetime = is_onetime || is_onetime_temp;
```

Comment:
It's not OK to provide multiple one-time integrated addresses as destinations. I think such cases are already rejected by the current mechanism for preventing use of multiple integrated addresses as destinations or use of explicit payment ID for a tx with an integrated address being one of the destinations.